theme/themepacks.less

```diff
@@ -45,6 +46,10 @@
     --pxt-toast-accent-info: #3182CE;
     --pxt-toast-accent-warning: #DD6B20;
     --pxt-toast-accent-error: #E53E3E;
+
+    // Eval dropdown

```

Comment:
I don't think the colors from the toasts work well for these dropdowns. 
<img width="97" alt="image" src="https://github.com/microsoft/pxt/assets/49178322/cdc37692-6730-4690-81ed-0c83fd780d2c">

When you say add variations, do you mean adding new variables but just specifying them as a toast color?

Reply:
These colors came from microbit, but were too strong for the toast, so we softened them by lowered their intensity. You can find the originals in the Figma, on the Components page: 
![image](https://github.com/microsoft/pxt/assets/12176807/34f5f5b5-9c19-4da5-9d32-6ba39e2b7247)

I believe these are what I used in the design of the results page. I wonder if these could be integrated into the theme as:

`--pxt-error` (red)
`--pxt-error-soft`  (red softened)
`--pxt-warning`  (yellow)
`--pxt-warning-soft`  (yellow softened)
`--pxt-success`  (green)
`--pxt-success-soft`  (green softened)
`--pxt-info`  (blue)
`--pxt-info-soft`  (blue softened)

Reply:
More consistent naming:

`--pxt-error` (red)
`--pxt-error-accent` (red softened)

Reply:
I'm unsure of the accent/soft use for us since dropdown doesn't use colors like that and we already have those "softened" colors in the toast coloring, but I can add these colors in as

`--pxt-warning`
`--pxt-error`
`--pxt-success`
`--pxt-info`

if that's preferred.

Reply:
Sorry, I should have mentioned... these would replace the toast color vars. Toast would used the the `-accent` vars.

Reply:
The toasts also have their own accents, though... Would we just keep those the same?
<img width="289" alt="image" src="https://github.com/microsoft/pxt/assets/49178322/e7713298-64b7-4421-94dc-c2accc3c641d">

Reply:
We should remove the toast vars entirely, rolling them into the warning/error/success/info categories. There will be background and accent variations of each, other variations if necessary. Whatever is needed to support toast and the result dropdown should be defined in these categories. It looks like accents will be necessary, so include those:

```
--pxt-warning-background
--pxt-warning-accent
```

If you'd like assistance working up the complete set of variable changes, please ping me on teams and we'll go through them.